ui/packages/catalog-ui-search/src/main/webapp/js/model/Query.js

```diff
@@ -478,6 +485,7 @@ Query.Model = PartialAssociatedModel.extend({
               })
             }
 
+            // TODO: Need to do something here?
             const srcStatus = result.get('status').get(search.src)
```

Comment:
This was a question for @andrewkfiedler. Nothing seems to be broken, but the key was changed from `status` to `statusBySource` on the `CqlQueryResponse`, this might be fine because we map that to just a list of `status` in the UI.